nirps/apero_raw_data_check/apero_raw_tests/tests/calib_test.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Blank example test do not modify this.
+
+Copy this to create your test
+
+Created on 2023-07-03 at 14:37
+
+@author: cook
+"""
+from typing import Any, Dict
+
+from apero_raw_tests.core import io
+from apero_raw_tests.core import misc
+from astropy.io import fits
+import os
+import glob
+import numpy as np
+from astropy.table import Table
+import copy
+
+# =============================================================================
+# Define variables
+# =============================================================================
+# define any other constants here that you want moving to the parameters.py
+#  file (these can be overwritten by the yaml file) and may change
+#  depending on the profile used (i.e. NIRPS_HA or NIRPS_HE)
+
+
+# =============================================================================
+# Define functions
+# =============================================================================
+def test(params: Dict[str, Any], obsdir: str, log=False) -> bool:
+    """
+    Test with observation directory exists
+
+    Passed = True
+
+    :param params: dictionary of parameters (from yaml/cmd line/ parameters.py)
+    :param obsdir: str, the observation directory (e.g. the night directory)
+    :param log: bool, if True prints messages (all messages must be wrapped
+                in a if log: statement)
+
+    :return: bool, True if passed, False otherwise
+    """
+    # define parameters we use here
+    raw_directory = params['raw dir']
+    # -------------------------------------------------------------------------
+    if log:
+        msg = 'Analysing observation directory: {0}'
+        margs = [obsdir]
+        misc.log_msg(msg.format(*margs), level='')
+    # -------------------------------------------------------------------------
+    # get all observation directories
+    obsdirs = io.get_obs_dirs(params)
```

Comment:
remove this line you don't use obsdirs

Reply:
also needs changing in eng_test.py

Actually as you use a mask make it a dictionary of numpy arrays (but this is what you do in line 100 any way)